test-suite/tests/compact-p002-out.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+  "@context": {
+    "ex": {"@id": "http://example.org/"}
+  },
+  "@id": "http://example.org/id1",
+  "@type": ["http://example.org/Type1", "http://example.org/Type2"],
+  "http://example.org/term": {"@id": "http://example.org/id2"}
+}
```

Comment:
@dlongley see comment [below](https://github.com/json-ld/json-ld.org/pull/511#issuecomment-312926594). Seems like a radical departure and totally breaks otherwise fine 1.0 contexts used for creating compact IRIs which may be more confusing, but if only done in a string 1.1 mode, that may be satisfactory. Let's hear some other opinions.